apps/notifications/src/panel/Notifications.jsx

```diff
@@ -136,6 +138,12 @@ export class Notifications extends PureComponent {
 		}
 	}
 
+	componentWillUnmount() {
+		const { customMiddleware } = this.props;
```

Comment:
One thing that would remove a lot of confusion would be renaming `customMiddleware` to something like `actionHandlers`. Because it's not Redux middleware at all, it's a set of handlers for the `actionMiddleware`.